framework/src/modules/token/stores/supported_tokens.ts

```diff
@@ -150,24 +161,30 @@ export class SupportedTokensStore extends BaseStore<SupportedTokensStoreData> {
 
 	public async removeSupportForToken(context: StoreGetter, tokenID: Buffer): Promise<void> {
 		if (this._isMainchainOrNative(tokenID)) {
-			return;
+			throw new Error('All tokens are supported.');
 		}
 		const allSupported = await this.allSupported(context);
 		if (allSupported) {
 			return;
```

Comment:
Here it needs to throw 'All tokens are supported', which you actually throw in the case above, where user tries to remove LSK or native token.